src/nl/surf/eduhub_rio_mapper/api.clj

```diff
@@ -100,19 +100,25 @@
                        ::ooapi/type type
                        ::ooapi/id   id))})))
 
+(defn link-route [{{:keys [rio-code]} :params :as request}]
+  (let [result   (job-route (assoc-in request [:params :action] "link"))
+        codename (if (= type "education-specifications") ::rio/opleidingscode ::rio/code)]
```

Comment:
`type` is not a parameter here, you're comparing the function `type` from clojure.core with a string.  Please add tests for all types to ensure this mistake does not happen again.

This is what I mean when I say our test coverage isn't good enough.  Errors like this should not happen!